index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <title>Amel Portfolio</title>
+</head>
+
+<body>
+  <header class="header-class">
+    <nav class="nav-container">
+      <div class="flex-logo"><img src="./img/My-Logo.png" alt="logo"></div>
```

Comment:
- [ ] Kindly use `a` tag instead of `div` because it is clickable and when you click logo it should redirect to homepage 🙏.

Reply:
Well done